games/t.yaml

```
@@ -635,7 +635,7 @@
 
 - name: Tux Builder
   originals:
-  - Super Mario
+  - SuperTux
```

```suggestion
  - Super Mario World
```
Since the game is extremely similar to SuperTux, which is in turn a game like Super Mario World, it's better to leave the original as SMW